src/ot_orchestration/dags/config/etl.conf

```diff
@@ -1,10 +1,1574 @@
-common.input = "gs://open-targets-pre-data-releases/24.06dev-test/input"
-common.output-base_path = "gs://open-targets-pre-data-releases/24.06dev-test/output"
+// --- Templated parameters --- //
 
-chembl_version = "34"
-ensembl_version = "112"
+common.input = "{{gcs_url}}/input"
```

Comment:
I am no expert to review this conf. Please get feedback from somebody from BE on this

Reply:
No need to review this, it's a copy of the original. The orchestration is happening here now, so the etl config belongs here.